File: Arrays/Python/Smaller_and_Larger.py
class Solution:

	def getMoreAndLess(self,arr, n, x):
		# code here
		less, greater = 0, 0
		for i in arr:
		    if i <= x:
		        less += 1
		    if i >= x:
		        greater += 1
        return less, greater

File: Solutions/Java/Smaller_and_Larger.java
class Solve {
    int[] getMoreAndLess(int[] arr, int n, int x) {
        // code here
        int count1 = 0, count2 = 0;
        
        for(int i: arr){
            if(x >= i) count1++;
            if(x <= i) count2++;
        }
        
        return  new int [] {count1, count2};
    }
}
